docs/reference/modules/db.md

```diff
@@ -0,0 +1,41 @@
+# `@apostrophecms/db`
+
+This module establishes `apos.db`, the mongodb driver connection object.
```

Comment:
`apos.db` is the "mongodb driver database instance." Otherwise too easily confused with `dbClient`.

Reply:
"Database object" would be OK too, for consistency with your note.